recipes-bsp/burn-boot/burn-boot_git.bb

```diff
@@ -13,7 +13,7 @@ do_install() {
     install -m 0755 ${S}/hisi-idt.py ${D}${bindir}
 }
 
-RDEPENDS_${PN} += "python python-pyserial"
+RDEPENDS_${PN} += "python3-core python3-pyserial"
```

Comment:
this is not sufficient, We see errors like 

```
ERROR: burn-boot-git-r0 do_package_qa: QA Issue: /usr/bin/hisi-idt.py contained in package burn-boot requires /usr/bin/python, but no providers found in RDEPENDS_burn-boot? [file-rdeps]

```

Reply:
@kraj yes I know we also need https://github.com/96boards-hikey/burn-boot/pull/1 merged to resolve that error.

@fboudra do you know who has merge rights on 96boards-hikey burn-boot repo?